database/migrations/2022_11_06_201538_create_training_waiting_list_account_pending_removal_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('training_waiting_list_account_pending_removal', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedInteger('waiting_list_account_id');
+            $table->enum('status', ['Pending', 'Completed', 'Cancelled'])->default('Pending');
+            $table->unsignedInteger('emails_sent')->default(0);
+            $table->timestamp('removal_date');
```

Comment:
```suggestion
            $table->timestamp('removed_at');
```
This isn't just a date but a datetime and should be named accordingly.

Reply:
I have renamed this to `remove_at `because it is a flag for when an account should be removed from a list (in the future).